config/ci.mocha.node.opts

```diff
@@ -0,0 +1,2 @@
+--reporter dot
```

Comment:
what does the 2 options do? Is it possible to add comments in the configuration file?

Reply:
I think we can. That will require us to upgrade mocha to the latest version (5.2.0) in all of our packages. We have 5.0.5 currently.

I can create another pull request to bump the mocha version in all of our SDK packages. Does this kind of change require approvals from all the SDK owners? Or is it okay to upgrade as long as all test continue to pass?

Reply:
yep, it would require approvals from them. Upgrading should be fine.

Reply:
Created #1375 to upgrade the mocha version. Feel free to add more reviewers who you think should be aware of this change to that pull request.

Reply:
Added comments in the file to explain the usage of these flags.